charts/incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,70 @@
+image:
+  repository: yandex/clickhouse-server
+  pullPolicy: IfNotPresent
+  tag: 21.3.20.1@sha256:4eccfffb01d735ab7c1af9a97fbff0c532112a6871b2bb5fe5c478d86d247b7e
+
+controller:
+  type: statefulset
+  strategy: RollingUpdate
+  rollingUpdate:
+    unavailable: 1
+
+securityContext:
+  capabilities:
+    add:
+      - IPC_LOCK
+      - NET_ADMIN
+      - SYS_NICE
+
+service:
+  main:
+    ports:
+      main:
+        port: 8123
+        targetPort: 8123
```

Comment:
Please keep the port type as HTTP.
It's not optional and should've been correctly set on all apps already...